src/main/java/com/github/fishio/control/AchievementScreenController2.java

```diff
@@ -0,0 +1,102 @@
+package com.github.fishio.control;
+
+
+
+
+import com.github.fishio.Preloader;
+import com.github.fishio.achievements.Achievement;
+import com.github.fishio.logging.Log;
+import com.github.fishio.logging.LogLevel;
+
+import javafx.fxml.FXML;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+
+/**
+ * Achievement screen controller.
+ */
+public class AchievementScreenController2 implements ScreenController {
+	
+	private Log log = Log.getLogger();
+	private Achievement playerDeath = new Achievement("playerDeath");
+	private Achievement enemyKill = new Achievement("enemyKill");
+	private Achievement hitWall = new Achievement("HitWall");
+	private Achievement revive = new Achievement("Revive");
+	private Achievement pun = new Achievement("Pun");
+
+	@FXML
+	private Button btnBackToMenu;
+	private Label bigachieve1;
```

Comment:
Correct, they will all need a @FXML annotation. Right now they are not used, but the next commit would make them used. The idea is that every picture has to be set on or off depending on whether their achievement is gained or not.

To be honest, I did not intend to add  ea41f25 and ea5dc06 to this pull request yet. However, I thought the pull request was already merged and therefore I pushed these two commits. I think for now it's best to just wait for my next commit which will actually replace the old Achievement screen with the next one.

<naming,review